src/app/views/user/user-settings/user-settings.component.scss

```diff
@@ -39,68 +38,69 @@
         }
 
         .file {
-          width: 85px;
+          cursor: pointer;
```

Comment:
Properties should be ordered width, height, cursor, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)